network/network.go

```diff
@@ -131,22 +136,30 @@ func (c *Collector) Describe(ch chan<- *prometheus.Desc) {
 // Collect implements the Collect method of the Collector interface.
 func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 	hosts := []string{}
+	c.scrapid++
+
+	scrapingStart := time.Now()
+	c.logger.Log("level", "info", "message", fmt.Sprintf("collecting metrics id: %d", c.scrapid))
```

Comment:
```suggestion
	c.logger.Log("level", "info", "message","collecting metrics", "scrapeid", c.scrapID))
```

IMO we should add `scrapeid` key to all the logs. Then we can do `| luigi -grep 'scrapeid=2'` and have all the logs for a Collect run filtered.

Reply:
That makes sense